src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -562,9 +563,10 @@ public async Task ReplyForbiddenWithWwwAuthenticateHeaderAsync(IEnumerable<strin
 
             string parameterString = string.Join(", ", parameters.Select(p => $"{p.Key}=\"{p.Value}\""));
 
-            if (CurrentHttpContext != null)
+            httpResponse ??= CurrentHttpContext?.Response;
+
+            if (httpResponse != null)
```

Comment:
if the httpResponse is null, do we want to throw with an aka.ms link?

Reply:
@jmprieur That was my initial proposal - throw an Exception, something like InvalidOperationException. I think just a descriptive message will suffice (ex. HttpContext is null, pass HttpResponse as a parameter). What's your idea of what to put into a wiki article about this?

Reply:
@pmaytak : Let's throw an InvalidOperationException with the message your proposed. Good idea!